### PR TITLE
Add import mnemonic screen

### PR DESCRIPTION
**Pull request by frm**
[Demo video of the new flow](https://www.loom.com/share/f9e79a52d6124458966ce6f4b4e1f074)

### CODE REVIEW

File: src/popup/components/Protocol/SetupScreen/index.tsx [hunk 1]
```diff
@@ -104,7 +115,109 @@ function ResetButton() {
...
+      </Subsubtitle>
+    </Container>
+  );
+}
+
+function Import({ onBack }: ImportProps) {
+  const dispatch = useUserDispatch();
+  const [mnemonic, setMnemonic] = useState<string | null>(null);
+  const [address, setAddress] = useState<string | null>(null);
+  const [disabled, setDisabled] = useState<boolean>(true);
```
Comment (shelbyd) [inline src/popup/components/Protocol/SetupScreen/index.tsx]: Prefer positive booleans. Much easier to reason about. You can negate it when used.

File: src/popup/components/Protocol/SetupScreen/index.tsx [hunk 2]
```diff
@@ -80,6 +85,12 @@ const CTA = styled.div`
   justify-content: center;
 `;
 
+const ClickableText = styled.span`
```
Comment (shelbyd) [inline src/popup/components/Protocol/SetupScreen/index.tsx]: This interferes with screen readers and keyboard navigation. Please have this be an <a> with no href.

**Review by shelbyd — COMMENTED** [review]
This differs from the design assets attached to the story. Is that intentional?

**Comment by shelbyd** [comment]
The UI is quite different from the ones in the provided screens. I think we should implement those screens instead of taking this as is.

**Comment by frm** [comment]
Reworked the UI: https://www.loom.com/share/17c334cf977e47228b91ce736ac20874

File: src/popup/components/Protocol/SetupScreen/index.tsx (hunk 2 above)
Comment (frm) [reply src/popup/components/Protocol/SetupScreen/index.tsx]: Empty anchors interfere with screen readers as well, as an example, `jsx-a11y` doesn't allow empty hrefs. I can turn it into a button and remove all the outline. Just leave the text, if you agree.

```
src/popup/components/Protocol/SetupScreen/index.tsx
  Line 242:9:  The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
```

Comment (shelbyd) [reply src/popup/components/Protocol/SetupScreen/index.tsx]: I guess a text button is the answer then.

File: src/popup/components/Protocol/SetupScreen/index.tsx [hunk 3]
```diff
@@ -104,7 +134,121 @@ function ResetButton() {
...
+      <HeaderWithLogo />
+
+      <Text
+        height={TextHeights.EXTRA_LARGE}
+        size={TextSizes.LARGE}
+        weight={TextWeights.BOLD}
+      >
+        You should only register a new identity if it isn't already associated
+        with an account. If you already have registered please recover your
+        account.
```
Comment (shelbyd) [inline src/popup/components/Protocol/SetupScreen/index.tsx:126]: For right now, I think we do want a different Substrate address for each install. That way, each install can see if it has registered for minting and not fail to extend another one's dataset.

I'll ask about changing this copy.

File: src/popup/components/Protocol/SetupScreen/index.tsx (hunk 2 above)
Comment (frm) [reply src/popup/components/Protocol/SetupScreen/index.tsx]: Fixed

**Comment by frm** [comment]
@shelbyd new flow: https://www.loom.com/share/981dc23f8f41489eb658b444af7f14fe

Let me know if this is what you had envisioned.

**Review by shelbyd — APPROVED** [review]

**Comment by shelbyd** [comment]
Looks perfect. :+1:

**Pull request merged**